executor/cleanup/cleanup.go

```diff
@@ -23,23 +23,20 @@ import (
 	"time"
 
 	apiv1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/fission/fission"
 	"github.com/fission/fission/crd"
-	"github.com/fission/fission/executor/fscache"
 )
 
 var (
 	deletePropagation = meta_v1.DeletePropagationBackground
 	delOpt            = meta_v1.DeleteOptions{PropagationPolicy: &deletePropagation}
 )
 
-// cleanupObjects cleans up resources created by old executortype instances
-func cleanupObjects(kubernetesClient *kubernetes.Clientset,
+// CleanupObjects cleans up resources created by old executortype instances
+func CleanupObjects(kubernetesClient *kubernetes.Clientset,
```

Comment:
You might want to make this more canonical go by taking the package name into account when naming your functions: `cleanup.Objects` instead of `cleanup.CleanupObjects`

Reply:
Rename package from `cleanup` to `reaper` instead.